_maps/shuttles/skyrat/pirate_nri_raider.dmm

```diff
@@ -17,12 +17,18 @@
 /obj/item/storage/box/zipties{
 	pixel_x = 2
 	},
-/obj/item/gun/energy/e_gun/advtaser,
-/obj/item/gun/energy/e_gun/advtaser,
-/obj/item/gun/energy/e_gun/advtaser,
 /obj/item/melee/baton,
 /obj/item/melee/baton,
 /obj/item/melee/baton,
+/obj/item/gun/energy/e_gun/advtaser{
+	w_class = 3
+	},
+/obj/item/gun/energy/e_gun/advtaser{
+	w_class = 3
+	},
+/obj/item/gun/energy/e_gun/advtaser{
+	w_class = 3
+	},
```

Comment:
Oh damn I forgot to remove this, that's not meant to be on this branch; but regarding the question, it's to make them fit into backpacks because otherwise they're counted as bulky items and, obviously, don't fit in.

Reply:
I feel like it wouldn't be so difficult to just, make a new subtype with the weight class set to normal, and then put it in here instead of a varedit.